sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1883,6 +1883,58 @@ class HiveDDLSuite
     }
   }
 
+  test("SPARK-26836: support Avro schema evolution") {
```

Comment:
Could you add an opposite test case which column removal schema evolution, please?

Reply:
`Avro` is known to support (1), (2), (3) from the following four.
```
 *   1. Add a column
 *   2. Hide a column
 *   3. Change a column position
 *   4. Change a column type (Upcast)
```

Reply:
It's 22:53 here so the rest of the requirements remains for the future.

Reply:
Sure, take your time~

Reply:
I have added a new unit test: `SPARK-26836: support Avro schema evolution (remove column)`